src/components/Alphabet/AlphabetPresentationSelected.js

```diff
@@ -130,7 +130,7 @@ function AlphabetPresentationSelected({
         <Link
           to={`/${sitename}/${
             kids ? 'kids/' : ''
-          }alphabet/startsWith?${CHAR}=${title}`}
+          }alphabet/startsWith?${CHAR}=${title}&types=word`}
```

Comment:
You could also use the constant `TYPE_WORD` here, in place of 'word' to future proof it.